Formula/urlview.rb

```diff
@@ -30,4 +30,12 @@ def install
                           "--sysconfdir=#{etc}"
     system "make", "install"
   end
+
+  test do
+    server = fork do
```

Comment:
I'm not that familiar with `urlview`, but my understanding is that it takes a filename or standard input. If that's the case, then the `test` block here should test that behavior (but still kill the process, since we don't actually want to select a URL).